07-HTML5-APIS/js/dragNdrop.js

```diff
@@ -0,0 +1,23 @@
+$(window).ready(function () {
+    var dropZone = document.getElementById('soltar');
```

Comment:
Why don't you use jQuery to select DOM elements? And add even listeners as well. This ensure more crossbrowser compatibility.